relay/paseo/src/weights/runtime_parachains_coretime.rs

```diff
@@ -61,39 +59,47 @@ impl<T: frame_system::Config> polkadot_runtime_common::identity_migrator::Weight
 	/// Proof: `Dmp::DownwardMessageQueues` (`max_values`: None, `max_size`: None, mode: `Measured`)
 	/// Storage: `Dmp::DownwardMessageQueueHeads` (r:1 w:1)
 	/// Proof: `Dmp::DownwardMessageQueueHeads` (`max_values`: None, `max_size`: None, mode: `Measured`)
-	/// Storage: `Identity::SuperOf` (r:0 w:100)
-	/// Proof: `Identity::SuperOf` (`max_values`: None, `max_size`: Some(114), added: 2589, mode: `MaxEncodedLen`)
-	/// The range of component `r` is `[0, 20]`.
-	/// The range of component `s` is `[0, 100]`.
-	fn reap_identity(r: u32, s: u32, ) -> Weight {
+	fn request_revenue_at() -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `7522 + r * (5 ±0) + s * (32 ±0)`
-		//  Estimated: `11037 + r * (7 ±0) + s * (32 ±0)`
-		// Minimum execution time: 176_411_000 picoseconds.
-		Weight::from_parts(182_882_369, 0)
-			.saturating_add(Weight::from_parts(0, 11037))
-			// Standard Error: 1_902
-			.saturating_add(Weight::from_parts(1_581_866, 0).saturating_mul(s.into()))
-			.saturating_add(T::DbWeight::get().reads(8))
-			.saturating_add(T::DbWeight::get().writes(6))
-			.saturating_add(T::DbWeight::get().writes((1_u64).saturating_mul(s.into())))
-			.saturating_add(Weight::from_parts(0, 7).saturating_mul(r.into()))
-			.saturating_add(Weight::from_parts(0, 32).saturating_mul(s.into()))
+		//  Measured:  `2848`
+		//  Estimated: `6313`
+		// Minimum execution time: 92_241_000 picoseconds.
+		Weight::from_parts(93_731_000, 0)
```

Comment:
These are the exact same as Polkadot, when they're re-run you should check them against the hardcoded weights in the CoretimeInterface and ensure there's a big enough buffer to allow runtime upgrades of one chain without the other.
It's something to keep an eye on and keep the hardcoded weights up to date, trying to maintain the size of the buffer. If there are changes in weight larger than the buffer, you'd have to upgrade the other chain first with a weight bump, which isn't ideal.